docs/cmdline-opts/ip-tos.md

```diff
@@ -0,0 +1,54 @@
+---
+c: Copyright (C) Daniel Stenberg, <daniel@haxx.se>, et al.
+SPDX-License-Identifier: curl
+Long: ip-tos
+Arg: <string>
+Help: Configure IP Type of Service (TOS) or Traffic Class
```

Comment:
```suggestion
Help: Set IP Type of Service or Traffic Class
```

We want this phrase to be as short and succinct as possible

Reply:
Done